ChangeLog.d/fix-rsa-performance-regression.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix
+   * Fix unintended performance regression when using short RSA public keys. See #9232.
```

Comment:
```suggestion
   * Fix unintended performance regression when using short RSA public keys. Fixes #9232.
```

A follow-up that improves testing is not relevant for users. The text in the changelog entry won't auto-close the issue, that only happens from the pull request description and commit comments.